includes/optional-modules/class-woo-member-commenting.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Woo member commenting.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Woo member commenting.
+ */
+class Woo_Member_Commenting {
+
+	/**
+	 * Initialize.
+	 *
+	 * @return void
+	 */
+	public static function init(): void {
+		add_action( 'init', [ __CLASS__, 'on_init' ] );
+	}
+
+	/**
+	 * Action callback on init.
+	 *
+	 * @return void
+	 */
+	public static function on_init() {
```

Comment:
In the past, we've included a snippet like this to automatically transition sites that have the standalone version of the plugin to the optional module instead (to avoid conflicts/collisions):

```suggestion
	public static function on_init() {
		// If the standalone plugin is active, deactivate it and activate as a module.
		if ( is_plugin_active( 'newspack-woocommerce-member-commenting/newspack-woocommerce-member-commenting.php' ) ) {
			deactivate_plugins( 'newspack-woocommerce-member-commenting/newspack-woocommerce-member-commenting.php' );
			Settings::activate_optional_module( 'woo-member-commenting' );
		}

```

Reply:
There are only 3 sites that use the old one. Since we are also changing the settings constant, I think it's safer to fix those 3 manually. Does that make sense?

Reply:
Makes sense, thanks for clarifying.